### ISSUE TITLE
Install multiple versions of /Applications/postgres.app for macOS

### ISSUE THREAD

**Issue by andrewchristopher** [PostgresApp/PostgresApp#703]
Hello.

I am currently running v13 of postgres.app installed to /Applications/Postgres.app on my mac. This is in accordance with installation instructions.

I would now like to install v15 whilst retaining v13. The installation instructions are the same as for v13, i.e. 
Download   ➜   Move to Applications folder   ➜   Double Click

As I already have postgres.app in /Applications for v13 I am struggling to see how this is going to work and I wish to ensure that my v13 database is preserved.

Will you please advise. Thank you.

**Comment by jakob** [comment]
1) Download Postgres.app with all supported versions

2) Quit Postgres.app

3) Replace old Postgres.app with the version you downloaded

4) Start new Postgres.app

Your v13 server will still be there

**Comment by andrewchristopher** [comment]
No. Doesn't work. V13 now doesn't start. 'Required PostgreSQL version not installed'.

**Comment by tbussmann** [comment]
make sure to use the "Postgres.app with all currently supported versions" download (first one under "Additional Releases" on the [downloads page](https://postgresapp.com/downloads.html))
